lib/admin/WP_Auth0_Admin_Advanced.php

```diff
@@ -780,7 +815,7 @@ public function link_accounts_validation( $old_options, $input ) {
 	 * @return array
 	 */
 	public function loginredirection_validation( $old_options, $input ) {
-		$new_redirect_url = strtolower( $input['default_login_redirection'] );
+		$new_redirect_url = esc_url_raw( strtolower( $input['default_login_redirection'] ) );
```

Comment:
why does this one get an `esc_url_raw` but the $old_redirect_url doesn't?

Reply:
We don't need to sanitize the old one to check it as it's already saved. If the new email is different after sanitization, then we update with that one (assuming it passes all the other checks).